shared/utils.ts

```diff
@@ -1,3 +1,15 @@
+const capitalizeString = (value: string) => {
```

Comment:
Should we install lodash and use `_.upperFirst()`?

Lodash will be useful anyway, I think

https://lodash.com/docs/4.17.15#upperFirst

Reply:
Much better indeed, thanks!